src/muya/lib/contentState/formatCtrl.js

```diff
@@ -104,6 +125,13 @@ const addFormat = (type, block, { start, end }) => {
   }
 }
 
+const checkTokenIsInlineFormat = token => {
+  const { type, tag } = token
+  if (FORMAT_TYPES.includes(type)) return true
+  if (type === 'html_tag' && tag === 'u') return true
```

Comment:
What  about HTML bold, italic etc? We could remove these tags also?

Reply:
According to my prediction, we should only add underline to the format box, and will not add any other html_tags, so there should be no more html bold italic etc. but we may add emoji to the float box later.

Reply:
haha, I think subscript and superscript also need to use html_tag syntax after I saw your comments.